### DISCUSSION TITLE
如何识别pag中文本和图片的对应关系

### DISCUSSION THREAD

**Discussion by fangjinxin** [Tencent/libpag#2164] [Q&A] [answered]
设计师提供了一个pag文件。
后端需要自动替换pag里面的图片和文字生成新视频。
但图片和文字有对应关系，
比如image1,image2 对应text1，image3 对应text2
这个后端怎么识别解决？

**Comment by kevingpqi123** [comment, marked as answer]
pag 中提供了图层名称获取的 API，可以在 AE 中针对特定的图层如文本图层、图片图层设置图层名称，导出 pag 文件后，终端可以通过 API 读取到图层名称，可以基于图层名称建立映射
